apps/test/unit/templates/instructions/teacherFeedback/TeacherFeedbackTest.jsx

```diff
@@ -53,7 +54,7 @@ describe('TeacherFeedback', () => {
 
   describe('viewed as Teacher - looking at student work', () => {
     describe('without previous feedback', () => {
-      it('does not display TeacherFeedbackStatus', () => {
```

Comment:
There are several renames here because when I renamed the components I didn't update the description of the tests